PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -101,10 +101,27 @@ public final class DefaultControlsViewController: ContentControlsViewController
         }
     }
     
-    var isApperared = false
+    private var controllerState = ControllerState()
+    struct ControllerState {
+        var hasAppeared: Bool = false
+        var isTransitioning: Bool = false
+        
+        var controlsAnimationPossible: Bool {
```

Comment:
Why `possible`? `allowed` sounds better for me.

Reply:
I thought that `possible`means that it is not possible to animate controls unless the view has appeared and that current layout is not forced by the transition.

Reply:
Ok, leave at is.